model/validation/DeliveryValidatorFactory.php

```diff
@@ -36,6 +36,11 @@ class DeliveryValidatorFactory extends ConfigurableService
                         'allow_punctuation' => true
                     ]
                 ]
+            ],
+            DeliveryAssemblyService::PROPERTY_START => [
```

Comment:
That's gonna be one tough backport there :)

IMO, for the backported version, it would be better to simply add that validator into `DeliveryForm` itself.

Reply:
yes, for backport it will be so

Reply:
Oh, in fact, I just realized the factory has bee introduced in https://github.com/oat-sa/extension-tao-delivery-rdf/releases/tag/v13.2.0.
And the current version of that extension for NCCER is https://github.com/oat-sa/extension-tao-delivery-rdf/releases/tag/v13.1.0.

So, let's port the change on top of https://github.com/oat-sa/extension-tao-delivery-rdf/releases/tag/v13.2.0 as 13.2.0.1, and then use that a bit newer version in the community and nccer package.